app/views/system/indices/index.scala.html

```diff
@@ -124,19 +125,26 @@
                     }
 
                     <small>
-                        @if(index.getRange.getBegin.isEqual(0L)) {
-                            Contains messages up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
-                            @if(index.getName.equals(deflectorTarget) && index.getInfo != null) {
-                                , @Tools.byteToHuman(index.getInfo.getStoreSizeBytes)
-                            }
+                        @if(index.getName.equals(deflectorTarget)) {
+                            Contains messages up to @DateHelper.readablePeriodFromNow(DateTime.now)
                         } else {
-                            Contains messages from @DateHelper.readablePeriodFromNow(index.getRange.getBegin) up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
-                            @if(index.getName.equals(deflectorTarget) && index.getInfo != null) {
-                                (@Tools.byteToHuman(index.getInfo.getStoreSizeBytes) /
-                                <span class="number-format" data-format="0,0">@index.getInfo.getDocuments.count</span> messages)
+                            @if(index.getRange.getBegin.isEqual(0L)) {
+                                Contains messages up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
+                            } else {
+                                Contains messages from @DateHelper.readablePeriodFromNow(index.getRange.getBegin) up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
+                                @if(index.getName.equals(deflectorTarget) && index.getInfo != null) {
```

Comment:
This branch will never be called (the index will never be a deflector at that point), and is also redundant, the size is also rendered afterwards.